### ISSUE TITLE
Ability to approve new users before they can login

### ISSUE THREAD

**Issue by scallister** [sct/overseerr#622]
**Is your feature request related to a problem? Please describe.**
Right now it appears Overseerr lets anyone login with their Plex account and begin using whatever default permissions are provided. Even if no permissions are selected by default this still lets them login and see recent requests from other users and existing movies / tv shows on the server.

**Describe the solution you'd like**
I think one of these would fix this:
- Be able to approve users once they've attempted to login
- Have a new permission for "logging in" or "viewing dashboard"

**Comment by samwiseg0** [comment]
> see recent requests from other users
This is not true. They can only see their own requests. 

I do no think people should need to be approved as that is done when you invite them to your server. You should not need to do that twice but the ability to disable users does makes sense.

**Comment by scallister** [comment]
> This is not true. They can only see their own requests.

Thank you for calling that out, that is good to know.

I completely agree that inviting them and then approving them again would be pointless.

When I was testing this myself I tried having the dummy accounts login without an invite and they were able to login with their plex credentials without an invite. That is specifically what felt odd to me 🤔.

**Comment by samwiseg0** [comment]
> When I was testing this myself I tried having the dummy accounts login without an invite and they were able to login with their plex credentials without an invite. That is specifically what felt odd to me 🤔.

In most cases Overseerr is intended to be used by your family/friends on your Plex server. In that context, it would make sense for all users to have access to the request system the admin set up for them. As it is today, there is no way to disable users/prevent users that are shared with your Plex server from accessing overseerr. 

I do see where having the ability to disable certain users would make sense. But I do not see where having to be invited to the server makes much sense?

Why would you want to prevent the users that you share your Plex server with from logging in to overseer?

**Comment by scallister** [comment]
> As it is today, there is no way to disable users/prevent users that are shared with your Plex server from accessing overseerr.

Ah, I see where I'm misunderstanding. The two other accounts I had tested access with already had access to my Plex server 🤦‍♂️.

I just retested with an account that I haven't shared access to Plex with yet and it works as expected:
![image](https://user-images.githubusercontent.com/32499763/104144917-6927c500-5382-11eb-90b2-eb9ffbbd4222.png)

@samwiseg0 thank you for your time and help in understanding this.

Closing this out since this feature makes no sense given the above.
